src/widgets/index/index.ts

```diff
@@ -221,7 +221,13 @@ const index = (props: IndexProps): Index => {
               instantSearchInstance: localInstantSearchInstance,
               state: helper!.state,
               templatesConfig: localInstantSearchInstance.templatesConfig,
-              createURL: localInstantSearchInstance._createAbsoluteURL!,
+              createURL: (nextState: SearchParameters) =>
```

Comment:
The function gets created at each render which wasn't the case previously.

Reply:
20f1e8d2de4e66ac7d2da04787cb81d70d39e076